test/utils.test.js

```diff
@@ -41,15 +43,30 @@ test('should take in a cartItem and return a tr element with the apporiate conte
     expect.equal(actual.outerHTML, expected);
   
 });
-////////////
+////////////    PASS
 test('should take in a cartItem and returns the total price for a single item', (expect) => {
-    const cartItemQuantity = 5;
+    const cartItemQuantity = 1;
     const cartItemPrice = 3;
         
-        
-    const expected = 15;
-    const actual = calculateItem(cartItemQuantity, cartItemPrice);
+    const expected = 3;
+    const actual = (cartItemQuantity, cartItemPrice);
 
     expect.equal(actual, expected);
   
 });
+//////////// PASS
```

Comment:
looks like tests are not yet passing but may be a syntax error

Reply:
Yes that one no  all other three of four  pass except the last one I keep getting innerhtml issue with that one and it throws off the entire test. But when that test is gone the rest pass not sure what is wrong. Also my pages are not showing up correctly and I am not sure what the problem is.